Source/Blazorise/Components/Row/RowContext.cs

```diff
@@ -9,10 +9,10 @@ public class RowContext : IRowContext
 {
     #region Members
 
-    private int spaceUsedByColumns = 0;
-
     private List<IColumnComponent> columns;
 
+    private Dictionary<Breakpoint, int> usedSpaces = new();
```

Comment:
Same allocation problem we talked bout earlier. If no columns, no used spaces. no need for aloc.

Reply:
done